README.md

```diff
@@ -1,14 +1,37 @@
 # docker-awscli
-Template repository for DataWorks GitHub
+Repository for `docker-awscli`
```

Comment:
we often put a link to the DockerHub repo here, something we could make part of the template